### DISCUSSION TITLE
Error message (#10002: Security error) appears if user selected 'PayPal Express Checkout' payment method

### DISCUSSION THREAD

**Discussion by aleksandrakorolova** [scandipwa/scandipwa#1622] [Q&A] [answered]
**Description**:
Error message 'Unable to place order: PayPal gateway has rejected request. Security header is not valid (#10002: Security error).' appears if user selected 'PayPal Express Checkout' payment method
**Expected behavior**:
Opens PayPal page 

**To Reproduce**:
1. Open website
2. Add product to cart
3. Open cart
4. Complete shipping step
5. Choose 'PayPal Express Checkout'
6. Agree with T&C
7. Tap 'Complete order'

**Actual result:**:
Error message 'Unable to place order: PayPal gateway has rejected request. Security header is not valid (#10002: Security error).' appears

**Comment by riha112** [comment, marked as answer]
Plugin is disabled for all ScandiPWA versions to enable it, requires adding line to ether local project or whole scandipwa repo in scandipwa.json file under extensions:
`"PayPal": "vendor/scandipwa/paypal-graphql"`
